iosApp/iosApp/Localizable.xcstrings

```diff
@@ -8884,6 +8884,47 @@
         }
       }
     },
+    "When using TalkBack, we can skip reading out maps to keep you focused on transit information." : {
```

Comment:
It does feel a little bit weird to have to add this string to iOS when it only actually exists in Android, but that's a natural consequence of having the source of truth live on one side specifically, and if we didn't add it here there'd be no way to mark it as needs review. (My initial plan had been to just swap in "TalkBack" for "VoiceOver" in the translated strings, but not all the translated strings even mention VoiceOver by name, so I've just gone with the usual approach instead.)